src/lib.rs

```diff
@@ -148,10 +133,10 @@ impl<'a> RRule<'a> {
         let mut until = "";
 
         // assign default weekstart and reassign if present
-        let mut wkst = "MO";
+        let mut _wkst = "MO";
```

Comment:
I plan to eventually add support for this, but for now this is right

Reply:
yeah, we can re-enable that at that time.